cve_scripts/paths2json.py

```diff
@@ -0,0 +1,34 @@
+import sys, getopt, os, json
+
+#Add the pathname fo the file that contains the pop paths data on your local machine
+#pathname = 
+filename = os.path.basename(pathname)
```

Comment:
im confused where this pathname is coming from?

Reply:
For this, I looked through Tristan's scripts to look for the output file of the pop paths data which I think happens in analyzer.py but I don't see an explicit output file.

Reply:
ok yeah we need to change this to where tristans pop path file is outputted. @TristanB22 where is that exactly?

Reply:
in host_scripts/ssh_command.sh -- " sudo scp -r -i $(cat ../directory.txt)/key -o StrictHostKeyChecking=no -P 2222 ubuntu@localhost:/home/ubuntu/GCOV_DATA $(cat ../directory.txt)/GCOV_DATA/$(date --iso-8601="minutes")"

When I was working on this script this command would sometimes block -- I was never able to figure out why exactly.

Reply:
Lol that formatted super strangely because of markdown but I think you know what I mean... /pop-paths-scripts/host_scripts/ssh_command.sh line 58

Reply:
@TristanB22 does your script generate an analyzer file? Thats what were looking for

Reply:
No, but that shouldn't be too difficult to create since we know the path to the data and just need to feed that to the analyzer. Was doing it manually before because we were just looking to get the data files instead of the analyzer files before.

Reply:
ok @TristanB22 will add this after we merge this PR